Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -634,6 +719,10 @@ public static void GetAssetNotification(Action<LootLockerPlayerAssetNotification
             LootLockerAPIManager.GetPlayerAssetNotification(onComplete);
         }
 
+        /// <summary>
+        /// Get asset deactivations since the last time this endpoint was called.
```

Comment:
Is this also for the player specifically?

Reply:
Yes 👍